Source/Core/AI/JobDrivers/JobDriver_DeliverFood.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using RimWorld;
+using Verse.AI;
+
+namespace PrisonLabor.Core.AI.JobDrivers
+{
+    public class JobDriver_JailorDeliverFood : JobDriver_FoodDeliver
```

Comment:
Jailor should not deliver food when prisoner can take food by himself.

Reply:
Right forgot that.

Reply:
i already fixed it